client/state/billing-transactions/actions.js

```diff
@@ -33,3 +36,26 @@ export const requestBillingTransactions = () => {
 			} );
 	};
 };
+
+export const sendBillingReceiptEmail = ( receiptId ) => {
```

Comment:
Ah, thanks for mentioning that. I'll skip this time, but I'll make sure to try when implementing new actions.